scripts/update/01-run.sh

```diff
@@ -54,6 +44,12 @@ if [[ ! -z "${UMBREL_OS:-}" ]]; then
         echo "ERROR: No Umbrel installation found at SD root ${SD_CARD_UMBREL_ROOT}"
         echo "Skipping updating on SD Card..."
     fi
+    
+    # Update apt packages on update
+    DEBIAN_FRONTEND=noninteractive apt-get install unattended-updates -y
+    # Manual run of the update (Normally for debugging purposes only, but we don't want to have a potential backdoor in Umbrel)
+    # https://wiki.debian.org/UnattendedUpgrades#Manual_run_.28for_debugging.29
+    unattended-upgrade -d
```

Comment:
It would be nice to update the status file here saying something like "Running security updates" so the user gets some feedback on this step, it could take a while.